treeherder/webapp/api/note.py

```diff
@@ -67,13 +78,33 @@ def create(self, request, project):
         """
         POST method implementation
         """
+        j = Job.objects.get(repository__name=project, id=int(request.data['job_id']))
+        fc_id = int(request.data['failure_classification_id'])
         JobNote.objects.create(
-            job=Job.objects.get(repository__name=project, id=int(request.data['job_id'])),
-            failure_classification_id=int(request.data['failure_classification_id']),
+            job=j,
+            failure_classification_id=fc_id,
             user=request.user,
             text=request.data.get('text', ''),
         )
 
+        if fc_id == 2:
```

Comment:
This needs a comment explaining id 2 means 'fixed by commit' (= backout, follow-up commit).